@nav-frontend/modal-styles/src/index.css

```diff
@@ -0,0 +1,54 @@
+.ReactModal__Body--open {
+  overflow: hidden;
+}
+.navds-modal {
+  background-color: var(--navds-color-white);
+  display: block;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: 4px;
+  position: relative;
+  flex-grow: 0;
+  overflow: auto;
+  max-height: 100%;
+  margin-bottom: 0;
+  z-index: 1010;
```

Comment:
Skal vi lage noe sånt for z-index? https://designsystem.morningstar.com/visual-language/constants/?version=3.0.5&tab=current#z-index

Reply:
God ide! Fikser det.